examples/nodejs/src/snippets/license/floating-license.ts

```diff
@@ -0,0 +1,40 @@
+import * as ScanbotSDK from "scanbotsdk";
+
+export class FloatingLicenseSnippet {
+  public static async initWithFloatingLicense(): Promise<void> {
+    const licenseKey = "SCANBOTSDK-LICENSE";
+    const writablePath = ".";
+
+    await ScanbotSDK.initialize({
+      licenseKey,
+      writablePath,
+    });
+
+    // Create device session with timeout (15 seconds)
+    const deviceSession = new ScanbotSDK.DeviceSession(15_000);
```

Comment:
magic constant

Reply:
I think we have a default timeout constant inside the DeviceSession

Reply:
Doesn't DeviceSession implement AsyncDisposable? I think it does, than you don't need the try-finaly, just await using deviceSession = ...